### PR TITLE
docs:add weekly report and 2022 commmunity summary

### PR DESCRIPTION
**Pull request by KID-G**
Changes:
Add weekly report and 2022 commmunity summart

### CODE REVIEW

File: blog/zh/blog/2023/01/02/2022-summary.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+author: "Sylvia"
+authorURL: "https://github.com/SylviaBABY"
+authorImageURL: "https://github.com/SylviaBABY.png"
+keywords: 
+- 社区动态
+- 年终总结
+- APISIX
+- 开源社区
+description: 本文介绍了 Apache APISIX 社区在 2022 年取得的成绩。
+tags: [Ecosystem]
```
Comment (SylviaBABY) [inline blog/zh/blog/2023/01/02/2022-summary.md]:
```suggestion
tags: [Community]
```

File: blog/zh/blog/2023/01/02/2022-summary.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+**产品快速成长的背后，离不开社区中每一位成员的支持。** 在过去的一年中，APISIX 的贡献者人数增加了 200 人左右。相比总贡献者人数，这一年的增长相当于总人数的近 40% 比例，可见今年 APISIX 收获了开发者们超高的热情反馈，并将对开源产品的喜爱体现在了社区贡献中来。
+
+这三年的发展过程中，APISIX 得到了非常多开发者的肯定与认可，他们也在全球 20 多个国家和地区进行 APISIX 项目的宣传。为了感谢各位对 APISIX 社区的喜爱，我们也会定期为参与 APISIX 社区的新晋贡献者们赠送社区周边，今年也是一样。
+
+上半年的新晋贡献者我们已派发过相关周边，下半年的新晋贡献者礼物也在紧锣密鼓地制作中。如果你有收到来自 APISIX 社区的相关邮件，记得填写信息哦。当然，你也可以主动在 [GitHub 页面](https://github.com/apache/apisix/blob/master/CONTRIBUTING.md)进行填写，之后就会有精美周边送到你手上。
+
+![contributor](https://static.apiseven.com/2023/01/03/63b3ea8cc7637.png)
+
+除了国内外开发者的认可外，今年 APISIX 也收获了很多企业级的技术赞誉。比如在互联网保险领域，众安保险在整个 DevOps 的全生命周期中，将 APISIX 作为流量治理的关键环节去处理，实现了更好的多渠道流量隔离；在智能汽车 AI 领域，地平线则选择了 APISIX Ingress 替换 Traefic 作为公司的统一流量网关，丰富了多云环境下的服务调用等功能；在国产手机领军队伍中，vivo 使用 APISIX 替换了之前传统的 NGINX，同时基于 APISIX 还自研了内部业务配置管理系统，在 vivo 业务架构的实践和场景支持上都表现得十分出色。
```
Comment (SylviaBABY) [inline blog/zh/blog/2023/01/02/2022-summary.md]: you can add links with these cases

vivo: https://apisix.apache.org/zh/blog/2022/11/13/vivo-with-apache-apisix/
地平线：https://apisix.apache.org/zh/blog/2022/10/28/apisix-ingress-with-horizon-ai/
众安保险：https://apisix.apache.org/zh/blog/2022/03/02/zhongan-usercase-with-apache-apisix/

File: blog/zh/blog/2023/01/02/2022-summary.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+同时在今年的海外探索中，我们也积极与学生群体进行交流，与伯克利、多伦多大学、滑铁卢大学等在美国、加拿大、印度、非洲的大学和组织共举办了 15 场 workshops，总量覆盖数百名 CS 和热爱开源的学生们。这也让 APISIX 的贡献者群体从公司的核心工程师扩展到了知名高校的大学生们，为后续的开源群体带来了国际化新鲜血液。得益于 workshop 的活跃，APISIX 社区的 Slack 渠道活跃人数也得到了巨大的增长。
+
+全球化的探索，也让 APISIX 收获了来自国际化企业的认可。来自土耳其最大的电子商务公司 Trendyol，在今年开始在生产环境中更换为 APISIX，期间也积极地在社区中进行相关进度反馈。与此同时，甲骨文 Oracle 在其 Spring Cloud 的 BaaS 产品中也开始使用 APISIX 来部署和保护示例应用程序的 API。
+
+![Global usercase](https://static.apiseven.com/2023/01/03/63b3eae00f8de.png)
+
+## 新的一年，新的展望
+
+在过去的时间里，Apache APISIX 先后获得云计算开源产业联盟评选的「尖峰开源项目及开源社区」、CSDN 机构评选的「年度云原生技术产品」等，并被 Stackshare 评为「2021 年全球 Top100 开发者工具」。非常感谢来自全球视角的用户肯定，新的一年，APISIX 社区会更加努力地为全世界用户提供更多的惊喜。
```
Comment (SylviaBABY) [inline blog/zh/blog/2023/01/02/2022-summary.md]:
```suggestion
在过去的时间里，Apache APISIX 先后获得云计算开源产业联盟评选的[「尖峰开源项目及开源社区」](https://mp.weixin.qq.com/s/YcCgbfbMTz7R3abHaI-p2Q)、CSDN 机构评选的[「年度云原生技术产品」](https://mp.weixin.qq.com/s/V40SxXoPxvHLIUIx4lxX-g)等，并被 [Stackshare 评为「2021 年全球 Top100 开发者工具」](https://mp.weixin.qq.com/s/CY4YH3JwQBiLeFHPGhyiaw)。非常感谢来自全球视角的用户肯定，新的一年，APISIX 社区会更加努力地为全世界用户提供更多的惊喜。
```

**Review by SylviaBABY — DISMISSED** [review]

File: blog/zh/blog/2023/01/02/2022-summary.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+除了国内外开发者的认可外，今年 APISIX 也收获了很多企业级的技术赞誉。比如在互联网保险领域，[众安保险](https://apisix.apache.org/zh/blog/2022/03/02/zhongan-usercase-with-apache-apisix/) 在整个 DevOps 的全生命周期中，将 APISIX 作为流量治理的关键环节去处理，实现了更好的多渠道流量隔离；在智能汽车 AI 领域，[地平线](https://apisix.apache.org/zh/blog/2022/10/28/apisix-ingress-with-horizon-ai/) 则选择了 APISIX Ingress 替换 Traefik 作为公司的统一流量网关，丰富了多云环境下的服务调用等功能；在国产手机领军队伍中，[vivo](https://apisix.apache.org/zh/blog/2022/11/13/vivo-with-apache-apisix/) 使用 APISIX 替换了之前传统的 NGINX，同时基于 APISIX 还自研了内部业务配置管理系统，在 vivo 业务架构的实践和场景支持上都表现得十分出色。
+
+![usercase](https://static.apiseven.com/2023/01/03/63b3ead0d9004.png)
+
+当然，以上也仅仅只选取了部分使用 APISIX 的企业用户，想要了解不同行业对于 APISIX 的实践，可以前往 [社区官网](https://apisix.apache.org/zh/blog/tags/case-studies/) 查看更多细节。
+
+## 活动形式多样，持续践行 Apache Way
+
+在 2022 年中旬，APISIX 社区举办了第一次全球性质的大会—— Apache APISIX Summit Asia 2022。在为期两天的活动中，我们见证了不同行业领域下的 APISIX 实践用例。由来自腾讯云、阿里云、VMvare Tanzu Labs 等顶级科技公司的工程师以及 APISIX PMC 成员们，带来了干货满满又富有借鉴意义的技术展示。
```
Comment (SkyeYoung) [inline blog/zh/blog/2023/01/02/2022-summary.md]:
```suggestion
在 2022 年中旬，APISIX 社区举办了第一次全球性质的大会—— Apache APISIX Summit Asia 2022。在为期两天的活动中，我们见证了不同行业领域下的 APISIX 实践用例。由来自腾讯云、阿里云、VMware Tanzu Labs 等顶级科技公司的工程师以及 APISIX PMC 成员们，带来了干货满满又富有借鉴意义的技术展示。
```

File: blog/zh/blog/2023/01/04/weekly-report-0105.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/2023/01/05/63b66c8388a6c.png
+---
+
+> 从 2022.12.19 - 2023.1.1，有 21 位开发者为 Apache APISIX 提交了 64 个 commits。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
```
Comment (SkyeYoung) [inline blog/zh/blog/2023/01/04/weekly-report-0105.md]:
```suggestion
> 从 2022.12.19 - 2023.1.1，有 21 位开发者为 Apache APISIX 提交了 64 个 commit。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
```

**Review by SkyeYoung — CHANGES_REQUESTED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**